lib/firmata.js

```diff
@@ -741,6 +750,14 @@ Board.prototype.sendString = function(string) {
   this.sp.write(data);
 };
 
+function i2cRequest(board, buffer) {
+
+  if (!i2cActive.get(board)) {
```

Comment:
Actually non-firmata implementations wouldn't even reach such code if they don't need config / init. So throwing the error for firmata.js makes the most sense then.